pontoon/base/templates/translate.html

```diff
@@ -326,7 +326,13 @@ <h3 class="loading"><div class="fa fa-refresh fa-spin"></div></h3>
           <button id="clear" title="Clear Translation (Ctrl + Shift + Backspace)">Clear</button>
           <button id="save" title="Save Translation (Enter)"></button>
         {% else %}
-          <p id="sign-in-required"><a id="sidebar-signin" href="{{ provider_login_url(request) }}">Sign in</a> to translate.</p>
+          <p id="sign-in-required">
```

Comment:
@jotes standalone login url do not accept next parameter, that causes error. I was asking, should I add support for `next` parameter in `standalone_login`? If not, I will change id for link and it will work same as in navigation.